.github/workflows/liquid.yml

```diff
@@ -7,7 +7,8 @@ jobs:
       matrix:
         entry:
           - { ruby: 2.5, allowed-failure: false } # minimum supported
-          - { ruby: 3.0, allowed-failure: false } # latest
+          - { ruby: '3.0', allowed-failure: false }
```

Comment:
There should be no reason to test intermediate versions if there isn't version dependent code.  Testing the minimum supported will make sure we don't use too new a feature and testing the latest version will help catch deprecations or feature removals.

```suggestion
```